packages/core/src/views/account-center/Minimized.svelte

```diff
@@ -40,15 +40,18 @@
       ? firstAccount.balance[firstAddressAsset]
       : null
 
-  $: primaryChain = primaryWallet.chains[0]
+  $: primaryChain =
+    primaryWallet && primaryWallet.chains && primaryWallet.chains.length
+      ? primaryWallet.chains[0]
+      : undefined
```

Comment:
```suggestion
  $: primaryChain =
    primaryWallet && primaryWallet.chains[0]
```
Since `primaryWallet.chains` is always an array I think we can skip the `.chains` and `.length` checks

Reply:
I'd be better this way

`primaryChain = primaryWallet?.chains?.[0]`

Reply:
@codingwithdidem thank you for the review and I agree the optional chaining is more clean but for wide range bundler support we are unable to include JS operators later than ES7